doc/classes/ProjectSettings.xml

```diff
@@ -432,19 +432,24 @@
 			If [code]true[/code], the home indicator is hidden automatically. This only affects iOS devices without a physical home button.
 		</member>
 		<member name="display/window/size/always_on_top" type="bool" setter="" getter="" default="false">
-			Force the window to be always on top.
+			Forces the main window to be always on top.
+			[b]Note:[/b] This setting is ignored on iOS, Android, and HTML5.
 		</member>
 		<member name="display/window/size/borderless" type="bool" setter="" getter="" default="false">
-			Force the window to be borderless.
+			Forces the main window to be borderless.
+			[b]Note:[/b] This setting is ignored on iOS, Android, and HTML5.
 		</member>
 		<member name="display/window/size/fullscreen" type="bool" setter="" getter="" default="false">
-			Sets the window to full screen when it starts.
+			Sets the main window to full screen when the project starts. On Windows, this will use exclusive fullscreen unless [member display/window/size/borderless] is [code]true[/code]. On Linux and macOS, this will use a borderless window since Linux and macOS don't have a concept of exclusive fullscreen.
```

Comment:
> this will use exclusive fullscreen unless

I do not think exclusive fullscreen is supported, it's always borderless-fullscreen (borderless window with the size of the screen).

> macOS, this will use a borderless window since Linux and macOS don't have a concept of exclusive fullscreen.

On macOS, it will start it on the new separate desktop (it's not the same as borderless window, and it's called "Full Screen" in all macOS menus, resolution won't be changed, it's the same effect as clicking green button without holding Option).

Reply:
@Calinou I guess the above feedback needs to make its way in the docs.